src/wasm/kotlin/java/arcs/wasm/WasmRuntimeClient.kt

```diff
@@ -0,0 +1,56 @@
+package arcs
+
+import arcs.wasm.*
```

Comment:
""

Reply:
It seems like intellij automatically substitutes explicit imports with *
I'll look into changing that setting. 
Is there a way we can persist such a preference across users in the project?